build.gradle

```diff
@@ -0,0 +1,131 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+    
+  dependencies {
+    // Needed for pre-compiling the JSPs
+    classpath "com.bmuschko:gradle-tomcat-plugin:${gradleTomcatPluginVersion}"
```

Comment:
Neat :+1: 
this is something uPortal looked at briefly but haven't had a chance to adopt yet.